components/suggestion/at_mention_provider/at_mention_suggestion.jsx

```diff
@@ -112,12 +112,22 @@ export default class AtMentionSuggestion extends Suggestion {
         } else {
             itemname = item.username;
 
-            if ((item.first_name || item.last_name) && item.nickname) {
-                description = `${Utils.getFullName(item)} (${item.nickname})`;
-            } else if (item.nickname) {
-                description = `(${item.nickname})`;
-            } else if (item.first_name || item.last_name) {
-                description = `${Utils.getFullName(item)}`;
+            const fullName =
+                item.first_name || item.last_name ? `${Utils.getFullName(item)}` : '';
+            const nickName = item.nickname ? `(${item.nickname})` : '';
```

Comment:
Some more nitpicking, but nickname shouldn't have the second N capitalized since it's only one word instead of two

Reply:
✅  Updated as per request/discussion